website/other/bezier-rs-demos/src/features/subpath-features.ts

```diff
@@ -1,6 +1,22 @@
 import { tSliderOptions } from "@/utils/options";
 import { ComputeType, SliderOption, SubpathCallback, WasmSubpathInstance } from "@/utils/types";
 
+const tErrorOptions = {
+	variable: "error",
+	min: 0.001,
+	max: 0.525,
+	step: 0.0025,
+	default: 0.02,
+};
+
+const tMinimumSeperationOptions = {
+	variable: "minimum_seperation",
+	min: 0.001,
+	max: 0.25,
+	step: 0.001,
+	default: 0.05,
+};
```

Comment:
Could you move these option definitions to the `utils/options.ts` file and give them more descriptive names such as `intersectionErrorOptions`